torch/_inductor/codegen/triton.py

```diff
@@ -1228,10 +1218,10 @@ def call_kernel(self, code, name: str):
         # TODO(jansel): if there are constants, we shouldn't bother passing them as args
         for tree in self.range_trees:
             if isinstance(tree.numel, (sympy.Integer, sympy.Symbol)):
-                expr = texpr(tree.numel)
+                expr = pexpr(tree.numel)
             else:
                 expr = f"{name}_{tree.prefix}numel"
-                code.writeline(f"{expr} = {texpr(tree.numel)}")
+                code.writeline(f"{expr} = {pexpr(tree.numel)}")
```

Comment:
I didn't audit spots where we currently call texpr and should call pexpr, trusting you to have gotten all of these

Reply:
In other places we import `pexpr` from `codegen.wrapper`, it has been updated.